waspc/cli/.dir-local.el

```diff
@@ -0,0 +1 @@
+((haskell-mode . ((dante-target . "--wasp"))))
```

Comment:
Whoops this should go into .gitignore. What happened to the same file that I have in /test/?